src/utils/projectV2.ts

```diff
@@ -263,7 +263,7 @@ export const getLocalizedPath = (path: string, locale: string): string => {
   }
 
   // If path already starts with locale, return as is
-  if (cleanPath.startsWith(`/${locale}/`)) {
+  if (cleanPath.startsWith(`/${locale}`)) {
```

Comment:
Couldn't this cause an issue?

What if the cleanPath starts with `/france-salesforce` which is a project slug?